views/index.ecr

```diff
@@ -16,10 +16,34 @@
 
     <section class="repos">
       <div class="container">
+        <div class="sorting">
+          Sort by:
+            <% {"stars", "updated", "forks"}.each_with_index do |target_sort, index| %>
```

Comment:
Sure! In fact we can have the sort options in a constant and use it here and in app.cr where we check the sort param is valid.